project_event/i18n/fr.po

```diff
@@ -276,6 +276,11 @@ msgstr "Catégorie d'activité d'événement"
 msgid "Event Activity Template"
 msgstr "Modèle d'activité d'événement"
 
+#. module: project_event
+#: model:ir.ui.view,arch_db:project_event.project_event_tree
+msgid "Event Number"
+msgstr "Numéro d'Évènement"
```

Comment:
@jananjoy Numéro d'événement

Reply:
@rimbendhaou : Update  the  translation to the fr.po file